### ISSUE TITLE
Write manuscript describing the results of this project.

### ISSUE THREAD

**Issue by goodb** [geneontology/pathways2GO#92]
Brainstorm document: https://docs.google.com/document/d/1QKVWxWkcbTh-A6VMzidusOS-CIP_SD1yEfAzWR5GiGs/edit

**Comment by goodb** [comment]
I think we should close the current github project and set up another one specifically on the topic of writing the manuscript.  
Here is another document with a fairly technical description of the conversion process.  https://docs.google.com/document/d/1x6ZfDuvLlqPC2vUSdf63fOu9j7IWK3jXUJ_JAL1bzsg/edit 

I think it or parts of it should be integrated into the main document.  

There is a lot of leftover stuff in the main document right now.  I think it would be useful to go ahead and start removing things that aren't going to stay.  I am assuming this falls to @deustp01 ? But if called upon I would be happy to try.

**Comment by deustp01** [comment]
I will work on the main document this week.

**Comment by goodb** [comment]
I started building the queries for making reports for the manuscript.  Table with data emerging here:  https://docs.google.com/spreadsheets/d/1OLMyc5gPsBDFNRDlIIN5BmVdQ3DaDwKxtXDt0Wk4uHs/edit?usp=sharing  

Let me know if there is something in particular you would like to measure.

**Comment by deustp01** [comment]
finished next round of revision of introduction and discussion.

**Comment by deustp01** [comment]
Finished another round of revision - I think we now have a coherent introduction - results - discussion draft. (I've moved all previous pieces to the end of the file, in case something essential gopt cut during revision.) It is definitely missing some pieces, like the meat-and-potatoes summary at the beginning of the discussion of what we have accomplished, but that will be straightforward to assemble.

What we really need now is fresh eyes** to read through the draft. Do we have a coherent description of the project and its rationale? Have we properly handled the PD - AF distinction and how it plays out in this project? Have we left out something crucial? Have we included something unimportant (my sense is that the draft is very long - shorter would be better, but maybe we need all this material)?

** @ukemi @thomaspd @huaiyumi @vanaukenk

**Comment by ukemi** [comment]
I agree. At this point the manuscript does seem a bit long. And, we still haven't added the concrete examples of the pathways yet. Discussion for tomorrow's call.
